From 4b2b4279dc2e2762ce40cef77958bc9df328c466 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 11 Oct 2019 17:31:02 -0700
Subject: [PATCH] TOOLS-2326 doc updates for move away from cr.joyent.us
 (interim Triton contribution guidelines update while transitioning away from
 Gerrit)

---
 CONTRIBUTING.md | 33 ++++++++++++++++-----------------
 README.md       |  3 ---
 2 files changed, 16 insertions(+), 20 deletions(-)

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
index c556364..2802c1a 100644
--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -13,19 +13,28 @@
 Thanks for using Triton and for considering contributing to it!
 
 tl;dr:
-- Triton repos do *not* use GitHub pull requests (PRs)! You'll be asked to
-  re-submit PRs to Gerrit. See below.
+- Some Triton repos use Joyent's Gerrit (https://cr.joyent.us) instead
+  of GitHub pull requests (PRs). *We are currently in the process of
+  moving to using GitHub PRs. Until that move is complete, if the repo
+  is listed at <https://cr.joyent.us/#/admin/projects/> then it is still
+  using Gerrit.*
 - Triton repos use both GitHub issues and internal-to-Joyent Jira projects for
   issue tracking.
 
 
 ## Code
 
-The Triton project uses Gerrit at [cr.joyent.us](https://cr.joyent.us) for code
-review of all changes. Any registered GitHub user can submit changes through
-this system. If you want to contribute a change, please see the [Joyent Gerrit
-user
-guide](https://github.com/joyent/joyent-gerrit/blob/master/docs/user/README.md).
+All changes to Triton project repositories go through code review. Currently
+that is via one of the following:
+
+- Joyent's Gerrit at [cr.joyent.us](https://cr.joyent.us), if the repo is
+  listed at <https://cr.joyent.us/#/admin/projects/>. Any registered GitHub user
+  can submit changes through this system. If you want to contribute a change,
+  please see the [Joyent Gerrit user
+  guide](https://github.com/joyent/joyent-gerrit/blob/master/docs/user/README.md).
+
+- GitHub PRs, at the usual place on each repo.
+
 If you're making a substantial change, you probably want to contact developers
 [on the mailing list or IRC](README.md#community) first. If you have any trouble
 with the contribution process, please feel free to contact developers [on the
@@ -41,16 +50,6 @@ Guidelines](https://github.com/joyent/eng/blob/master/docs/index.md). Notably:
 * All repositories should be "make check" clean at all times.
 * All repositories should have tests that run cleanly at all times.
 
-"make check" checks both JavaScript style and lint.  Style is checked with
-[jsstyle](https://github.com/davepacheco/jsstyle).  The specific style rules are
-somewhat repo-specific.  See the jsstyle configuration file or `JSSTYLE_FLAGS`
-in Makefiles in each repo for exceptions to the default jsstyle rules.
-
-Lint is checked with
-[javascriptlint](https://github.com/davepacheco/javascriptlint). Repos sometimes
-have repo-specific lint rules, but this is less common -- look for
-"tools/jsl.node.conf" for per-repo exceptions to the default rules.
-
 
 ## Issues
 
diff --git a/README.md b/README.md
index 2afbd11..f5348de 100644
--- a/README.md
+++ b/README.md
@@ -75,9 +75,6 @@ Community discussion about Triton DataCenter happens in two main places:
 
 * In the *#smartos* IRC channel on the [Freenode IRC network](https://freenode.net/).
 
-You can also follow [@SmartDataCenter](https://twitter.com/SmartDataCenter) on
-Twitter for updates.
-
 
 ## Getting Started
 
-- 
2.17.2 (Apple Git-113)

